x/mongo/driver/serverapioptions.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package driver
+
+// ServerAPIOptions represents options used to configure the API version sent to the server
+// when running commands.
+type ServerAPIOptions struct {
+	ServerAPIVersion  string
+	Strict            *bool
+	DeprecationErrors *bool
+}
+
+// ServerAPI creates a new ServerAPIOptions configured with default values. ServerAPIVersion is set
+// to latest stable version.
+func ServerAPI() *ServerAPIOptions {
+	return &ServerAPIOptions{ServerAPIVersion: "1"}
```

Comment:
Why are we setting a default of "1" here? My understanding is that the Client construction code in the `mongo` package will validate that the server API version is non-empty and is supported by the driver, so by the time we get here, we should rely on that package to pass down the version directly.

Reply:
Yeah... Removed. I was doing this for the test files that were creating new inserts to have a constructor with a default version that won't cause an error, but those no longer have appended `ServerAPIOptions`.